template/jest.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   preset: 'react-native',
+  setupFilesAfterEnv: ["@testing-library/react-native/extend-expect"],
```

Comment:
I'd rather recommend creating `jest-setup.ts` file and putting the `import "@testing-library/react-native/extend-expect"` there. With typical RN app users quickly find need to add additional global mocks for popular community libraries.